apps/opik-backend/src/main/java/com/comet/opik/domain/TraceDAO.java

```diff
@@ -1174,7 +1176,7 @@ public Mono<ProjectStats> getStats(@NonNull TraceSearchCriteria criteria) {
             ST statsSQL = newFindTemplate(SELECT_TRACES_STATS, criteria);
 
             var statement = connection.createStatement(statsSQL.render())
-                    .bind("project_id", criteria.projectId());
+                    .bind("project_ids", List.of(criteria.projectId()).toArray(UUID[]::new));
```

Comment:
Nit: I believe the `toArray` call isn't necessary, as the ClickHouse client is able to interpret Java lists.

Other occurrences in this PR.

Reply:
Done